bosh/ruby-app-release/unit-tests/spec/helpers.rb

```diff
@@ -0,0 +1,48 @@
+def check_block_formations(rendered_template)
+
+  line_index = 0
+  tokens = Array.new #acts like a stack
+  exec_line = rendered_template.each_line do |line|
+    if line.include? "case"
+      tokens.push("case")
+    elsif line.match? (/.+\)/)
+      tokens.push("open")
+    elsif line.include? ";;"
+      tokens.push("close")
+    end
+  end
```

Comment:
Can't we make this more Ruby like? I honestly don't really understand the logic (or what it does), but I'm pretty sure that we can improve it :D